src/casp/services/api/data_manager/cell_quota.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+
+import sqlalchemy as sa
+
+from casp.data_manager import BaseDataManager
+from casp.services import settings
+from casp.services.db import models
+
+
+class CellQuotaDataManager(BaseDataManager):
+
+    # SQL query directory
+    SQL_QUERY_DIR = f"{settings.SERVICES_DIR}/api/data_manager/sql_queries"
+
+    # SQL queries from files
+    with open(f"{SQL_QUERY_DIR}/get_cells_processed_this_week_for_user.sql.sa", "r") as f:
+        SQL_GET_CELLS_PROCESSED_THIS_WEEK = f.read()
+
+    def get_remaining_quota_for_user(self, user: models.User) -> int:
```

Comment:
What do you think about placing this method under the CellQuotaService, since it is responsible for quota calculation rather than retrieving information from the database?

Reply:
I tried that initially, but got a not-fully-loaded-module/circular-import error.  I assume it was because `CellOperationsService` calls that method, so I had to import `CellQuotaService` in `CellOperationsService` and then both are also imported by the `services` module?  Although I wonder if maybe the import order caused the issue?  I'm not sure.  Any thoughts?